packages/tentales/src/middleware/__tests__/default-hooks.test.ts

```diff
@@ -0,0 +1,11 @@
+import { getDefaultHooks } from "../default-hooks"
+
+test("It gets public default hooks", async () => {
+  const resp = getDefaultHooks({ isPublic: true })
+  expect(resp).toMatchSnapshot()
```

Comment:
So looking at the snapshots, this will only really check that the middleware names are consistent and the object structure. Are snapshots really a good match for this use-case?

Reply:
So yeah, it will only really check that the middleware names are consistent and the object structure is the same. I think it's a balance, as we're basically testing a constant list here.